src/Illuminate/Pagination/AbstractCursorPaginator.php

```diff
@@ -512,152 +491,34 @@ public static function viewFactory()
     }
 
     /**
-     * Get an iterator for the items.
-     *
-     * @return \ArrayIterator
-     */
-    public function getIterator(): Traversable
-    {
-        return $this->items->getIterator();
-    }
-
-    /**
-     * Determine if the list of items is empty.
-     *
-     * @return bool
-     */
-    public function isEmpty()
-    {
-        return $this->items->isEmpty();
-    }
-
-    /**
-     * Determine if the list of items is not empty.
+     * Render the paginator using the given view.
      *
-     * @return bool
+     * @param  string|null  $view
+     * @param  array  $data
+     * @return \Illuminate\Contracts\Support\Htmlable
      */
-    public function isNotEmpty()
+    public function render($view = null, $data = [])
```

Comment:
I moved up the `render` method here, from the `CursorPaginator` child class. Changes nothing, but makes sure the tests are passing.